src/workerd/jsg/jsg.c++

```diff
@@ -416,23 +416,19 @@ kj::Arc<const ExternalMemoryTarget> Lock::getExternalMemoryTarget() {
 }
 
 kj::String Lock::accountedKjString(kj::Array<char>&& str) {
-  size_t size = str.size();
-  return kj::String(str.attach(getExternalMemoryAdjustment(size)));
+  return kj::String(kj::mv(str));
```

Comment:
Just add a comment here as a reminder for me to clean these up soon

```suggestion
  // TODO(Cleanup): The memory accounting that was attached to these strings
  // has been removed because it was too expensive. We should rethink how
  // to handle it. Making this non-ops for now and will remove the actual
  // methods separately.
  return kj::String(kj::mv(str));
```